backend/src/plugins/config.ts

```diff
@@ -90,7 +96,10 @@ const configPlugin: FastifyPluginAsync = async (server) => {
     EP7_TOKEN_VGL: process.env.EP7_TOKEN_VGL ?? '90000',
     EP7_TOKEN_PGL: process.env.EP7_TOKEN_PGL ?? '150000',
     EPV_06: process.env.EPV_06?.split(',') ?? ['0x5FF137D4b0FDCD49DcA30c7CF57E578a026d2789'],
-    EPV_07: process.env.EPV_07?.split(',') ?? ['0x0000000071727De22E5E9d8BAf0edAc6f37da032']
+    EPV_07: process.env.EPV_07?.split(',') ?? ['0x0000000071727De22E5E9d8BAf0edAc6f37da032'],
+    DELETE_KEY_RECOVER_WINDOW: parseInt(process.env.DELETE_KEY_RECOVER_WINDOW || '7'),
```

Comment:
@ch4r10t33r using 7 days because only 7-30 days values are allowed for scheduled deletion.